src/EventStore.ClientAPI/EventStore.ClientAPI.csproj

```diff
@@ -1,7 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
-		<TargetFrameworks>netstandard2.0;netcoreapp3.1</TargetFrameworks>
+		<TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">netstandard2.0;netcoreapp3.1;net452</TargetFrameworks>
```

Comment:
Hi @shaan1337 - hope you had a good Christmas period.
Not sure if you're aware of https://www.nuget.org/packages/Microsoft.NETFramework.ReferenceAssemblies/ but it allows you to target `net452` and build under Linux etc

Reply:
Hi @megakid, i'm fine, hope you're doing well. thanks for the tip! Yup, I wasn't aware that this was possible - I'll make the changes asap. thank you!